### DISCUSSION TITLE
Loadout bugs out when playing on vehicle maps

### DISCUSSION THREAD

**Discussion by rapkha** [Joe91/fun-bots#232] [Q&A] [answered]
Im not quite sure if this happens because of the mod or VU, since i cant seem to find any threads relating to this or any troubleshooting and the only conclusion i could draw was that the mod was acting up in some way. If i overlooked something or messed something up, sorry for wasting your time. Im new to VU and would greatly appreciate any leads i can follow to solve this. But anyways, the issue i am having is that when me and others play on vehicle maps (CQL(CQS/Assault maps that have bot support), there is a chance of the player loadout not showing up/loading in, ie. no guns or gadget (except for med and ammo packs) can be used and models for these are absent. Only way around is to wait a couple of minutes and then it would work again, but sometimes it wouldnt load in at all. I primarily tested this while only running fun-bots with no other mods running on my 64 player server, but the issue still persisted.

It usually happens randomly when playing on maps that include vehicles, especially when using them, but may also happen when playing infantry on those maps. Also has a tendency to happen when getting revived by bots or when picking up kits. Happens most frequently when the round is about halfway or almost over. A common indicator for a loadout bug is when vehicle textures turn to a lower resolution.

The current version used is fun-bots Release 2.5.3, but i have also experienced this on fun-bots Release 2.4.0.

VU versions: 
Former: Build 18848
Current: Build 19002

**Comment by Joe91** [comment, marked as answer]
Hi,
this is a known issue of VU or even frostbite, that happens with large playercount (and only on some gamemodes). That's the reason you usually only see it with bots. But I tried to improve some things with the spawning on fun-bots and the VU guys also changed some things. Is it still happening with the last release of fun-bots?
Sorry for the very late response by the way...

**Comment by Digitalinsite** [comment]
I had this happen too when I was experimenting with 40 bots per team - it seemed to happen less when I had only 20 per team.

Not a solution but a workaround is to hit the console key then type disconnect and then reconnect to the server and your gun etc comes back. Obviously you lose your score but better than having to kill the server. 

Awesome bot mod by the way. Hanging out to see what you have in store for funbots in the future!

**Comment by rapkha** [comment]
Hello
Thanks for the feedback and sorry for not responding sooner.
Ill be testing it out for 2.6.2 in the coming few days and ill give some feedback about it once i've gathered enough playtime
